.clang-format

```diff
@@ -0,0 +1,43 @@
+﻿# Suricata settings as per 
+# https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Coding_Style
```

Comment:
So this refers to the wiki's code-style, but there is also the code-style.rst in git that I have updated in this pull request. Ideally, the wiki page gets generated from the code-style.rst in here so it should not mattter, but I don't know if that is the case.

Question: Which one should I refer to in the .clang-format file?
1) wiki page
2) code-style.rst
3) none of the above (I don't like that option, but it's an option)

Reply:
`code-style.rst`. We only recently started dev related things to the sphinx docs

Reply:
Will be fixed in v5